src/Parser/HolyPriest/TALENT_DESCRIPTIONS.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+
+export default {
+  descriptions: {
```

Comment:
Oh, whoops. I was mimicking disc priest's setup for this since some of our talents are shared.